saleor/settings.py

```diff
@@ -522,17 +522,33 @@ def get_url_from_env(name, *, schemes=None) -> str | None:
 AZURE_CONTAINER_PRIVATE = os.environ.get("AZURE_CONTAINER_PRIVATE")
 AZURE_SSL = os.environ.get("AZURE_SSL")
 
+# Replicate behavior of creating default values
+STORAGES = {
+    "default": {"BACKEND": "django.core.files.storage.FileSystemStorage"},
+    "staticfiles": {"BACKEND": "django.contrib.staticfiles.storage.StaticFilesStorage"},
+}
+
 if AWS_STORAGE_BUCKET_NAME:
+    # removed in favor of STORAGES in django 5.1
```

Comment:
I think we can remove `STATICFILES_STORAGE` and `DEFAULT_FILE_STORAGE` entirely, the comment also won't be necessary since `STORAGES` is expected configuration in this version of Django.